src/test/java/stringCalculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,64 @@
+package stringCalculator;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+class StringCalculatorTest {
+
+    private StringCalculator stringCalculator;
+
+    @BeforeEach
+    void setUp() {
+        stringCalculator = new StringCalculator();
+    }
+
+    @DisplayName("빈 문자열 또는 null을 입력할 경우, 0을 반환해야 한다.")
+    @ParameterizedTest
+    @NullAndEmptySource
+    void emptyOrNull(final String text) {
+        assertThat(stringCalculator.add(text)).isZero();
+    }
+
+    @DisplayName("숫자 하나를 문자열로 입력할 경우, 해당 숫자를 반환한다.")
+    @ParameterizedTest
+    @ValueSource(strings = {"1"})
+    void onlyOneNumber(final String text) {
+        assertThat(stringCalculator.add(text)).isSameAs(Integer.parseInt(text));
```

Comment:
객체 지향 생활 체조 원칙 - `규칙 4: 한 줄에 점을 하나만 찍는다.`를 실행해보는건 어떨까요~?^^

```suggestion
        assertThat(stringCalculator.add(text))
                .isSameAs(Integer.parseInt(text));
```